core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -319,16 +389,26 @@
     margin-top: 0;
     padding: 0;
   }
+  .checklist-container {
+    padding: 10px;
+  }
   .chapter-card-inputs {
     padding: 30px 55px 30px 45px;
   }
+  .checklist-items .todo-item-container {
+    display: flex;
+    margin-bottom: 10%;
+  }
+  .checklist-items .todo-item-container span {
```

Comment:
Prefer using a class name instead of "span".

Reply:
Done